### ISSUE TITLE
Possibility to set the print size

### ISSUE THREAD

**Issue by jablonskipj** [artiebits/pdf-to-printer#69]
**Describe the solution you'd like**
I have a problem with the size of the printed label on the Zebra printer. I have a label that has to be 10x15 and when I print it manually out of the pdf file it is printed correctly (10x15), but when I have printed the same pdf using the library the end result is 70% of the destination size.

**Describe alternatives you've considered**
I was thinking about the printer configuratiin but I have ended up with the solution that was acceptable.

**Additional context**
Maybe we could provide the printing size via additional params that we are able to pass to the `print` function?

**Comment by artiebits** [comment]
Hi @jablonskipj, thanks for reporting the issue. There is a big discussion about how to implement options https://github.com/artiebits/pdf-to-printer/pull/19 but we still didn't find a proper solution. Any ideas? ;)

**Comment by artiebits** [comment]
Hi @jablonskipj, i've published `pdf-to-printer@1.1.0-beta.0` version with the solution that should solve your problem https://github.com/artiebits/pdf-to-printer/pull/70. Maybe the solution is not that good but can be improved later.

**Comment by jablonskipj** [comment]
@artiebits thank you for the answer. I will take a look at this week, so I will be able to give feedback soon :)

**Comment by jablonskipj** [comment]
@artiebits I have one question. do you want me to make the Code Review if the solution will be working?

**Comment by artiebits** [comment]
Hi @jablonskipj, please do if you want to review and give suggestions. But installing this beta version into your project and saying if it works or not is enough as well :). Thank you!

**Comment by artiebits** [comment]
Hi @jablonskipj, I've merged PR #70, if it solves your problem then I think this issue can be closed. Thanks!

**Comment by jablonskipj** [comment]
@artiebits great, thanks!

**Comment by jablonskipj** [comment]
I have to reopen the issue. I have used both `win32: ['-print-settings "2x"']` and `win32: ['-print-settings "fit"']` and it does not work for the Zebra printer. After I execute the `print` method with these options the printer does not change the size of the resulting print, but the function also throws info to the catch callback

```
C:\\node-printer-server\\node_modules\\pdf-to-printer\\dist\\SumatraPDF.exe -print-to-default -print-settings \"2x\" -silent C:\\Users\\admin\\Downloads\\t.pdf
```

or

```
```
C:\\node-printer-server\\node_modules\\pdf-to-printer\\dist\\SumatraPDF.exe -print-to-default -print-settings \"fit\" -silent C:\\Users\\admin\\Downloads\\t.pdf
```
```

**Comment by artiebits** [comment]
Sorry to hear this. Could you please download SumatraPDF.exe v3.0.0 (the version we currently use) and try to run it with `-print-settings "2x"` and `-print-settings "fit"` and see if it works. Also, I would suggest trying SumatraPDF.exe version v3.0.1, if it solves the problem then we can upgrade the version. Thank you.

**Comment by artiebits** [comment]
not sure if it helps but I found that when I implemented options I made a mistake, I was passing options to `execFile` in the wrong format. I released version `1.1.1-beta.1` with the fix. Unfortunately, I don't have access to a Windows machine anymore. Maybe you can try this version and say if it helps... 

Sorry for these issues!

**Comment by hakverir** [comment]
Hi, I'm trying on windows with version 1.1.1 and it doesn't work with both '-print-settings "2x"' and '-print-settings "fit"'. I've tried with SumatraPDF.exe v3.0.0, 3.1 and 3.1.1
